product_docs/docs/pgd/5/durability/group-commit.mdx

```diff
@@ -124,46 +124,44 @@ does eventually COMMIT even though the client might receive an abort message.
 
 See also [Limitations](limitations). 
 
-### Transaction Reconciliation
+### Transaction reconciliation
 
-A group commit transaction’s commit on the origin node is implicitly converted 
-into a two phase commit. 
+A Group Commit transaction's commit on the origin node is implicitly converted 
+into a two-phase commit. 
 
-In the first phase (prepare) the transaction is prepared locally and made ready to commit.
-The data is made durable but uncomitted at this stage so other transactions
-cannot see the changes made by this transaction. This prepared transaction gets
+In the first phase (prepare), the transaction is prepared locally and made ready to commit.
+The data is made durable but is uncomitted at this stage, so other transactions
+can't see the changes made by this transaction. This prepared transaction gets
 copied to all remaining nodes through normal logical replication. 
 
-The origin node seeks confirmations from other nodes, as per rules in the group
-commit grammar. If it gets confirmations from minimum required nodes in the
-cluster, it decides to commit this transaction moving onto the second phase (commit) 
-where it also sends this decision via replication to other nodes which will also eventually commit on getting this message.
+The origin node seeks confirmations from other nodes, as per rules in the Group
+Commit grammar. If it gets confirmations from the minimum required nodes in the
+cluster, it decides to commit this transaction moving onto the second phase (commit). 
+In the commit phase, it also sends this decision by way of replication to other nodes. Those nodes will also eventually commit on getting this message.
```

Comment:
Please check these edits, as it was a little hard to follow.

Reply:
I guess because of its Latin roots...I think this is one I discussed with Dee Dee and we decided not to use it. It's a pretty typical style guide thing, although I don't think we lose any clarity. It's like e.g. and i.e....perfectly understandable, just a style guide choice.